File: Fibonacci.java
import java.util.Scanner;

public class Fibonacci {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.println("Welcome to Fibonacci Series Generator!");
        System.out.print("Enter the number of terms: ");
        int numTerms = scanner.nextInt();

        int firstTerm = 0, secondTerm = 1;

        System.out.print("Fibonacci Series: " + firstTerm + " " + secondTerm);

        for (int i = 3; i <= numTerms; i++) {
            int nextTerm = firstTerm + secondTerm;
            System.out.print(" " + nextTerm);
            firstTerm = secondTerm;
            secondTerm = nextTerm;
        }

        scanner.close();
    }
}

File: Fibonacci.php
<?php

$n=0;
$a=0;
$b=1;
echo "Fibonacci series upto 15: ";
echo "$a,$b";

while($n<16)
    {
        $c=$b + $a;
        echo " , ";
        echo "$c";
        $a=$b;
        $b=$c;
        $n=$n+1;
    }

?> 